packages/core/admin/admin/src/content-manager/hooks/useFetchContentTypeLayout/index.js

```diff
@@ -68,10 +68,10 @@ const useFetchContentTypeLayout = contentTypeUID => {
   }, [contentTypeUID, getData]);
 
   const updateLayout = useCallback(
-    newLayout => {
+    data => {
       dispatch({
         type: 'UPDATE_LAYOUT',
-        newLayout: formatLayouts({ contentType: newLayout, components: {} }, schemas),
```

Comment:
I'm wondering if the problem could be solved here by reusing the previously received components info as they won't have changed 🤔